pkgs/tools/system/netdata/default.nix

```diff
@@ -1,36 +1,67 @@
-{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, makeWrapper
-, CoreFoundation, IOKit, libossp_uuid
-, nixosTests
-, bash, curl, jemalloc, json_c, libuv, zlib, libyaml, libelf, libbpf
-, libcap, libuuid, lm_sensors, protobuf
-, go, buildGoModule, ninja
-, withCups ? false, cups
-, withDBengine ? true, lz4
-, withIpmi ? (!stdenv.isDarwin), freeipmi
-, withNetfilter ? (!stdenv.isDarwin), libmnl, libnetfilter_acct
-, withCloud ? false
-, withCloudUi ? false
-, withConnPubSub ? false, google-cloud-cpp, grpc
-, withConnPrometheus ? false, snappy
-, withSsl ? true, openssl
-, withSystemdJournal ? (!stdenv.isDarwin), systemd
-, withDebug ? false
-, withEbpf ? false
-, withNetworkViewer ? (!stdenv.isDarwin)
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  bash,
+  buildGoModule,
+  cmake,
+  cups,
+  curl,
+  darwin,
+  freeipmi,
+  go,
+  google-cloud-cpp,
+  grpc,
+  jemalloc,
+  json_c,
+  libbpf,
+  libcap,
+  libelf,
+  libmnl,
+  libnetfilter_acct,
+  libossp_uuid,
+  libuuid,
+  libuv,
+  libyaml,
+  lm_sensors,
+  lz4,
+  makeWrapper,
+  ninja,
+  nixosTests,
+  openssl,
+  pkg-config,
+  protobuf,
+  snappy,
+  systemd,
+  withCloud ? false,
+  withCloudUi ? false,
+  withConnPrometheus ? false,
+  withConnPubSub ? false,
+  withCups ? false,
+  withDBengine ? true,
+  withDebug ? false,
+  withEbpf ? false,
+  withIpmi ? (stdenv.isLinux),
+  withNetfilter ? (stdenv.isLinux),
+  withNetworkViewer ? (stdenv.isLinux),
+  withSsl ? true,
+  withSystemdJournal ? (stdenv.isLinux),
+  zlib,
 }:
-
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   version = "1.47.0";
   pname = "netdata";
 
   src = fetchFromGitHub {
     owner = "netdata";
     repo = "netdata";
-    rev = "v${version}";
-    hash = if withCloudUi
-      then "sha256-eMapy4HQaEblDfZt2uVSfBWRlkstX7TxZDBgSv5bW/I="
+    rev = "v${finalAttrs.version}}";
```

Comment:
```suggestion
    rev = "v${finalAttrs.version}";
```

Reply:
lol ty... i hate when my cmp does that...

Reply:
Nice to know I'm not the only one